DatadogRUM/Sources/RUMConfiguration.swift

```diff
@@ -111,6 +111,13 @@ extension RUM {
         /// Default: `false`.
         public var trackBackgroundEvents: Bool
 
+        /// Determines whether the SDK should track application termination by the watchdog.
+        ///
+        /// Read more about watchdog terminations at https://developer.apple.com/documentation/xcode/addressing-watchdog-terminations
+        ///
+        /// Default: `false`.
+        public var trackWatchdogTermination: Bool
```

Comment:
convention/ Other `track*` options names are plural:
- trackBackgroundEvent**s**
- trackFrustration**s**

For consistency this one could be too:
```diff
- trackWatchdogTermination
+ trackWatchdogTerminations
```

Reply:
sure.